.github/workflows/tests.yml

```diff
@@ -20,18 +20,28 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      # Installing node, so we can install markdownlint next
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
```

Comment:
Gee it sure would be nice to have a linter that did not depend on a completely different language.

What about pymarkdownlint? 
https://pypi.org/project/pymarkdownlint/